src/engine/controls/bpmcontrol.cpp

```diff
@@ -161,39 +161,67 @@ double BpmControl::getBpm() const {
 }
 
 void BpmControl::slotAdjustBeatsFaster(double v) {
-    mixxx::BeatsPointer pBeats = m_pBeats;
-    if (v > 0 && pBeats && (pBeats->getCapabilities() & mixxx::Beats::BEATSCAP_SETBPM)) {
+    if (v <= 0) {
+        return;
+    }
+    TrackPointer pTrack = getEngineBuffer()->getLoadedTrack();
```

Comment:
```suggestion
    const TrackPointer pTrack = getEngineBuffer()->getLoadedTrack();
```

Reply:
I have changed this, but note that it makes only the address held by the TrackPointer const not the whole track.

Reply:
I know. I think you change the data that is pointed to, so making the data also const wouldn't work anyway.